### PR TITLE
Connect to Bookstore API

### PR DESCRIPTION
**Pull request by Draxeytina**
- [x] Use [axios] for making API calls.
- [x] Fetch book list from Bookstore API on Books page load.
- [x] Add an Action Creator, that returns a function.
- [x] Fetch the list of books.
- [x] Dispatch another action (add an action type) that will pass API response to reducer and update the state.
- [x] Refactor add book and remove book features to persist changes in the server.
- [x] Change existing Action Creators, so they return functions.
- [x] Make changes by using API.
- [x] Dispatch an action that updates the state accordingly.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @Draxeytina, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have proper README documentation, kindly use the [microverse template](https://github.com/microverseinc/readme-template) to create your README file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Draxeytina** [comment]
> Hi @Draxeytina,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that doesn't have proper README documentation, kindly use the [microverse template](https://github.com/microverseinc/readme-template) to create your README file
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Noted, thank you.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
+# Math Magicians
+A Single Page App (SPA) for that manages a book library from the Bookstore API
+
```
Comment (RolandM99) [inline README.md]: - [ ] Kindly, make sure that you write the correct title and description of your project.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
...
+- My coding partners
+- The Microverse code reviewers
+
+## 📝 License
+
+This project is [MIT](https://github.com/Draxeytina/Bookstore/MIT.md) licensed.
+
 # Getting Started with Create React App
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (RolandM99) [inline README.md]: - [ ] As you're already customized your README file as requested by the previous reviewer, I'll suggest you delete all about creating React App after the LICENSE section in your README file.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @Draxeytina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md]: This change has been done well

File: README.md (hunk 2 above)
Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Draxeytina , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Draxeytina** [comment]
> Hi @Draxeytina ,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

thank you @peterrobert

**Pull request merged**